src/redux/adhocRpcEndpoints.ts

```diff
@@ -0,0 +1,33 @@
+import { Address } from "@superfluid-finance/sdk-core";
+import {
+  RpcEndpointBuilder,
+  getFramework,
+} from "@superfluid-finance/sdk-redux";
+
+export const adhocRpcEndpoints = {

```

Comment:
I think the endpoint here is a redux toolkit terminology.

It can be easily confused with the endpoint of rpc gateway we often talk about in other context